### DISCUSSION TITLE
Vote here on additional languages for FreeSewing

### DISCUSSION THREAD

**Discussion by joostdecock** [freesewing/freesewing#4956] [Forum]
Below is a list of languages that have been suggested by our users.
If you want to vote for one of these, please use the 👍 emoji reaction on the comment the language links to.

- [Swedish](https://github.com/freesewing/freesewing/discussions/4956#discussioncomment-6989699) (Suggested by Victor Beck)

If you'd like to suggest another language, please [use this form to suggest a language that is not in the list above](https://next.freesewing.org/translation/suggest-language).

**Comment by joostdecock** [comment]
# Swedish

Give this comment a 👍 to vote for Swedish

**Comment by GalitU** [comment]
Russian will be helpful too
